CRM/Utils/Check/Component/Env.php

```diff
@@ -268,39 +269,50 @@ public function checkLastCron() {
     $statusPreference->domain_id = CRM_Core_Config::domainID();
     $statusPreference->name = 'checkLastCron';
 
+    $level = \Psr\Log\LogLevel::INFO;
+    $now = gmdate('U');
+
+    // Get timestamp of last cron run
     if ($statusPreference->find(TRUE) && !empty($statusPreference->check_info)) {
-      $lastCron = $statusPreference->check_info;
-      $msg = ts('Last cron run at %1.', [1 => CRM_Utils_Date::customFormat(date('c', $lastCron))]);
+      $msg = ts('Last cron run at %1.', [1 => CRM_Utils_Date::customFormat(date('c', $statusPreference->check_info))]);
     }
+    // If cron record doesn't exist, this is a new install. Make a placeholder record (is_active=0).
     else {
-      $lastCron = 0;
-      $msg = ts('No cron runs have been recorded.');
+      $statusPreference = CRM_Core_BAO_StatusPreference::create([
+        'name' => 'checkLastCron',
+        'check_info' => $now,
+        'is_active' => 0,
```

Comment:
should this not be set to `is_active = 1` and instead use the date expiry part of the check otherwise for new installs how will this be renabled?

Reply:
Oh, yes I see you're right it's a problem. I'll pick a different field.